src/contactgroupmodel.cpp

```diff
@@ -333,6 +333,7 @@ QHash<int,QByteArray> ContactGroupModel::roleNames() const
     roles[BaseRole + StartTime] = "startTime";
     roles[BaseRole + Groups] = "groups";
     roles[BaseRole + DisplayNames] = "displayNames";
+    roles[BaseRole + SubscriberIdentity] = "subscriberIdentity";
```

Comment:
that could come from the EVENT_PROPERTY_SUBSCRIBER_ID define in constants.h I guess, but probably not necessary / symmetric.